<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>NAT GAS WINDOW (NOT IMPLEMENTED)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QComboBox" name="hydroComboBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>501</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="hydroCustomerBox">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>0</y>
          <width>541</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="hydroBillTableWidget">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>30</y>
          <width>541</width>
          <height>461</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Service</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Amount</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
       </widget>
       <widget class="QTableWidget" name="hydroTableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>501</width>
          <height>461</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Provider</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Rate</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QComboBox" name="inetComboBox">
          <property name="placeholderText">
           <string>Select a service</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="inetTableWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNatGas"/>
   <addaction name="actionInternetService"/>
   <addaction name="actionHydro"/>
  </widget>
  <action name="actionNatGas">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/natural-gas.png</normaloff>:/icons/resources/icons/natural-gas.png</iconset>
   </property>
   <property name="text">
    <string>Nat Gas</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionInternetService">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/cloud-network.png</normaloff>:/icons/resources/icons/cloud-network.png</iconset>
   </property>
   <property name="text">
    <string>Internet Service</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionHydro">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/elec.png</normaloff>:/icons/resources/icons/elec.png</iconset>
   </property>
   <property name="text">
    <string>Hydro</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
